doc/multi-o.md

```diff
@@ -0,0 +1,117 @@
+# Multi Orchestrator Setup
+
+This document describes the various ways an orchestrator could run multiple nodes and separate concerns. 
```

Comment:
For terminology, I suggest we use "orchestrator operator" to refer to the entity running orchestrator nodes. So, this sentence would be updated to: "an orchestrator operator could run multiple nodes...".

Reply:
Could be specific about the node type, eg "an operator can run multiple orchestrator nodes."

I'd also elaborate a bit on the "separate concerns", probably in a follow-on sentence. What concerns are those? Wallet and key management? Blockchain monitoring? Submitting transactions?

Reply:
How about we refer to the role as "Orchestrator"  and the nodes as "orchestrator nodes" , wouldn't that be simple yet understandable enough?

Reply:
"This document describes the various ways an orchestrator could run multiple orchestrator nodes".

vs.

"This document describes the various ways an operator could run multiple orchestrator nodes".

The second option reads a bit smoother to me - I think it makes a clearer distinction between the entity running the nodes and the node type so I have a slight preference for this option.